recipes-support/provisioning/files/phytec-board-config.sh

```diff
@@ -40,7 +40,7 @@ calc_wt_size() {
     # NOTE: it's tempting to redirect stderr to /dev/null, so supress error
     # output from tput. However in this case, tput detects neither stdout or
     # stderr is a tty and so only gives default 80, 24 values
-    WT_HEIGHT=18
+    WT_HEIGHT=20
```

Comment:
Is it necessary to change the height with this PR? I guess there are not  any UI changes that would require this.

Reply:
i did some minor UI changes in the text. Maybe i should have done it in 2 separate commits since it has nothing to do with the restart of the awsclient.